client/my-sites/stats/stats-download-csv-upsell/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { isEnabled } from '@automattic/calypso-config';
+import page from '@automattic/calypso-router';
+import { Button, Gridicon } from '@automattic/components';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+
+interface Props {
+	className: string;
+	statType: string;
+	siteId: number;
+	borderless: boolean;
+}
+
+const StatsDownloadCsvUpsell: React.FC< Props > = ( {
+	className,
+	statType,
+	siteId,
+	borderless,
+} ) => {
+	const translate = useTranslate();
+
+	const onClick = ( event: React.MouseEvent< HTMLButtonElement, MouseEvent > ) => {
+		event.preventDefault();
+
+		const source = isEnabled( 'is_running_in_jetpack_site' ) ? 'jetpack' : 'calypso';
```

Comment:
Thinking of making `'jetpack'`, `'calypso'`, isEnabled( 'is_running_in_jetpack_site' ) a const since it's been showing up quite commonly.